src/backoffice/core/components/workspace-property/workspace-property.element.ts

```diff
@@ -90,7 +91,7 @@ export class UmbWorkspacePropertyElement extends UmbLitElement {
 	 * @default ''
 	 */
 	@property({ type: Object, attribute: false })
-	public set config(value: DataTypePropertyPresentationModel[]) {
+	public set config(value: UmbDataTypePropertyCollection) {
 		this._propertyContext.setConfig(value);
```

Comment:
I dont think this should be changed. This would still be of the previous type. As you didn't change the value parsed to this element.
That is the raw data given to the context, and first turned into a UmbDataTypePropertyCollection when send to the property.
So this should be reverted.